be/src/exec/vectorized/parquet_scanner.cpp

```diff
@@ -233,7 +233,7 @@ Status ParquetScanner::convert_array_to_column(ConvertFunc conv_func, size_t num
         auto nullable_column = down_cast<NullableColumn*>(column.get());
         auto null_column = nullable_column->mutable_null_column();
         size_t null_count = fill_null_column(array, _batch_start_idx, num_elements, null_column, _chunk_start_idx);
-        nullable_column->set_has_null(null_count != 0);
+        nullable_column->set_has_null(nullable_column->has_null() || null_count != 0);
```

Comment:
when null_count is 0, but nullable_column->has_null is true?

Reply:
Give some comment for that case

Reply:
```
size_t fill_null_column(const arrow::Array* array, size_t array_start_idx, size_t num_elements, NullColumn* null_column,
                        size_t column_start_idx) {
    null_column->resize(null_column->size() + num_elements);
    auto* null_data = (&null_column->get_data().front()) + column_start_idx;
    size_t null_count = 0;
    for (size_t i = 0; i < num_elements; ++i) {
        auto is_null = array->IsNull(array_start_idx + i);
        null_data[i] = is_null;
        null_count += is_null;
    }
    return null_count;
}
```

fill_null_column means append elements. if the first batch has null, but the second doesn't has null. 
the final result should has null

Reply:
please rename the `fill_null_column ` to `append_null_column`.

and rename the `const arrow::Array*` and `const ColumnPtr& column` to src_array, and dest_column